docs/en/stores/introduction.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+Dojo stores is a predictable, consistent state container with built-in support for common patterns.
+
+This package provides a centralized store designed to be the single source of truth for an application. It operates using uni-directional data flow. This means all application data follows the same lifecycle, ensuring the application logic is predictable and easy to understand.
+
+| Feature                   | Description                                                          |
+| ------------------------- | -------------------------------------------------------------------- |
+| global data store         | application state is stored globally in a single source of truth     |
```

Comment:
```suggestion
| global data store         | application state gets stored globally in a single source of truth     |
```

again I would somehow look to make it clear that this is global to the app and not a JS global without adding a bunch of words